src/figuresClasses.ts

```diff
@@ -1,11 +1,69 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0 || a >= b + c || b >= a + c || c >= b + a) {
```

Comment:
In the constructor of the `Triangle` class, the parameters `a`, `b`, and `c` should be accessed using `this.a`, `this.b`, and `this.c` instead of directly using `a`, `b`, and `c`. This is necessary because you are checking the instance properties, not the constructor parameters directly.